### PR TITLE
Added LWIMIQ entry

### PR DESCRIPTION
**Pull request by romanblanco**

### CODE REVIEW

**Comment by JPrause** [comment]
@syncrou @imtayadeway @jrafanie could each of you review this week's blog entry for today's merge.
Thanks for the entry @romanblanco

**Comment by imtayadeway** [comment]
@romanblanco I see the dates for your links are for last week's issue - you may be able to save some effort by generating a new blog post with `bin/miq generate lwimiq` and copying/pasting what you need from there.

**Comment by JPrause** [comment]
@imtayadeway @syncrou @jrafanie can you each review this week's blog entry.

File: site/_posts/2017-03-30-mostly-harmless-update.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+tags: LWIMIQ
+---
+
+Another week is behind us, a new one is here and it brings along the next
+weekly update. I'm [Roman](https://github.com/romanblanco) from ManageIQ UI
+team, and I'll walk you through what happened in the last week in ManageIQ.
+
+## Featured
+
+Attention please, numbers are comming!
```
Comment (JPrause) [inline site/_posts/2017-03-30-mostly-harmless-update.md]: I think you probably wanted to remove this line.

File: site/_posts/2017-03-30-mostly-harmless-update.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+This week we had:
+
+- 56 PRs in the [manageiq][manageiq PRs merged] repo
+- 54 in the [manageiq-ui-classic][manageiq-ui-classic PRs merged] repo
+- 13 in the [manageiq-ui-service][manageiq-ui-service PRs merged] repo
+
+It's woth mentioning we now have 3 merged Pull Requests
```
Comment (JPrause) [inline site/_posts/2017-03-30-mostly-harmless-update.md]: worth

File: site/_posts/2017-03-30-mostly-harmless-update.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+some issues are introduced. This week,
+[Martin](https://github.com/martinpovolny) has helped to keep the
+ManageIQ UI nice by fixing issue with missing metrics for
+Containers ([#819](https://github.com/ManageIQ/manageiq-ui-classic/pull/819))
+and missing table name on VMDB Summary page
+([#806](https://github.com/ManageIQ/manageiq-ui-classic/pull/806)).
+
+## New
+
+Of cource, we have also added a lot of new code.
```
Comment (JPrause) [inline site/_posts/2017-03-30-mostly-harmless-update.md]: course

**Comment by JPrause** [comment]
Nice job @romanblanco

**Comment by syncrou** [comment]
@romanblanco - This is really great! - I enjoyed hearing about the PR's merged from the students at Mendel University.

**Comment by JPrause** [comment]
@romanblanco have you made the edits recommended. If not,..please do, and then we can merge. Thanks again for writing the blog!

**Pull request merged**